horizomer/utils/tests/test_tree.py

```diff
@@ -327,39 +328,41 @@ def test_build_taxdump_tree(self):
                              '(19,20)8)3)1;'])
         self.assertTrue(compare_topology(obs, exp))
 
-    # test node ordering in increase and decrease orders
+
+    # test node ordering in increase and decrease orders using is_node_ordered
+    # function
     def test_order_nodes(self):
         """Test order nodes"""
-        tree1 = TreeNode.read(['(((a,b),(c,d,i)),((e,g),h));'])
-        tree1_ordered = order_nodes(tree1, True)
-        for node in tree1_ordered.postorder():
-            if node.is_tip():
-                node.n = 1
-            else:
-                node.n = sum(x.n for x in node.children)
-        prev = tree1_ordered.root().n
-        for node in tree1_ordered.levelorder():
-            if not node.is_tip():
-                cur = node.n
-                self.assertTrue(prev >= cur)
-                prev = cur
+        tree1 = TreeNode.read(['(((a,b),(c,d,i)j),((e,g),h));'])
+        # test increase ordering
+        tree1_increase = order_nodes(tree1, True)
+        self.assertTrue(is_nodes_ordered(tree1_increase))
```

Comment:
`assertTrue` is the right method here! You apparently got the secret of Python unit-test!